docs/data/material/components/tables/ReactVirtualizedTable.tsx

```diff
@@ -86,7 +86,9 @@ const VirtuosoTableComponents: TableComponents<Data> = {
   Scroller: React.forwardRef<HTMLDivElement>((props, ref) => (
     <TableContainer component={Paper} {...props} ref={ref} />
   )),
-  Table: (props) => <Table {...props} style={{ borderCollapse: 'separate' }} />,
+  Table: (props) => (
+    <Table {...props} style={{ borderCollapse: 'separate', tableLayout: 'fixed' }} />
```

Comment:
It might be better without inline-styles:

```suggestion
    <Table {...props} sx={{ borderCollapse: 'separate', tableLayout: 'fixed' }} />
```